lib/tasks/fix_fdw_credentials.rake

```diff
@@ -0,0 +1,35 @@
+desc 'Change FDW server and user credentails accroding to dot env file'
+task fix_fdw_credentials: :environment do
+  fix_primary_server_fdw_credentials
+  fix_remote_server_fdw_credentials
+end
+
+def fix_primary_server_fdw_credentials
+  config = ActiveRecord::Base.configurations['secondbase'][Rails.env]
+  server = 'fis'
+  active_record_execute(alter_fdw_server_query(server, config))
+  active_record_execute(alter_fdw_user_query(server, config))
+end
+
+def fix_remote_server_fdw_credentials
+  config = ActiveRecord::Base.configurations[Rails.env]
+  server = 'openhub'
+  second_base_execute(alter_fdw_server_query(server, config))
+  second_base_execute(alter_fdw_user_query(server, config))
+end
+
+def alter_fdw_server_query(server_name, config)
+  "ALTER SERVER #{server_name} OPTIONS(set host '#{config['host']}', set dbname '#{config['database']}', set port '#{config['port']}')"
+end
+
+def alter_fdw_user_query(server_name, config)
+  "ALTER USER MAPPING FOR #{config['username']} SERVER #{server_name} OPTIONS(SET password '#{config['password']}')"
```

Comment:
We should first create development database followed by test, by this way we won't face any issues. 

```
rake db:create
rake db:structure:load
rake db:migrate
rake fix_fdw_credentials

RAILS_ENV=test rake db:create
RAILS_ENV=test rake db:structure:load
RAILS_ENV=test rake db:migrate
RAILS_ENV=test rake fix_fdw_credentials
```